blog/_posts/btcpay-server-1-2-0.md

```diff
@@ -0,0 +1,139 @@
+
+### **Wallet Signing Wizard** 🧙
+
+BTCPay Server wallet allows you to receive and spend transactions. If you're self-hosting a server, it's a wallet relying on your full node. With hardware wallet support and an abundance of privacy-enhancing features, we've seen an increasing amount of wallet usage. The community feedback was that signing options could be improved.
+
+That's why the 1.2.0 now offers a signing wizard, which should simplify and, in some cases, automate choices that you may face when signing a transaction. If you're using a hot wallet, the wallet now signs instantly.
+
+Another change relating to the hot wallet, is that from now on, your hot wallet enables *PayJoin* by default.
+
+![BTCPay 1.2.0 Crowdfund view](/images/BTCPay1.0.2WalletSign.gif)
+
+### Apps UI Improved 💻
+
+The *Crowdfund* and *Point-Of-Sale* app creation pages have been updated - making navigation and creation of your favorite apps smoother.
+
+![BTCPay 1.2.0 Crowdfund view](/images/crowdfunding-ui-improvements.gif)
+
+![BTCPay 1.2.0 POS view](/images/pos-ui-improvements.gif)
+
+A number of UI improvements and fixes also made it into this release, namely relating to the following pages:
+
+* The server settings
+* The payment request
+* The notifications
+
+### **BTCPay Vault notification** 🔐
+
+BTCPay Vault is an application which allows you to plug your hardware wallet and use it within your BTCPay Server - thanks to the [HWI](https://github.com/bitcoin-core/HWI) (Hardware Wallet Integration).
+Due to changes in [HWI 2.0.1](https://github.com/bitcoin-core/HWI/releases/tag/2.0.1), BTCPay Server will now detect the version of the Vault application and notify you to update in case you're using an older version.
+
+For the full list, check out the [changelog](https://github.com/btcpayserver/btcpayserver/releases)
+
+## Top-up invoices 🧾
+
+Lots of our community members use BTCPay Server to accept donations. Unlike our regular invoice, where a fixed price and rate are needed, the donations don't have that requirement.
+
+Another interesting use-case is topping up the account. Certain merchants wanted to have the ability to accept whatever the amount is paid since they're crediting it to the customer's account. So whatever customer sends them, that's what they'll credit to their account.
+
+*Top-up invoice* is a new type of invoice, which doesn't require an amount. This invoice is considered paid as soon as payment is detected.
+
+![BTCPay 1.2.0 Crowdfund view](/images/topupinvoice.png)
+
+For now, the top-up invoice is available in the [pay button](https://docs.btcpayserver.org/FAQ/FAQ-Apps/#what-is-a-payment-button). Depending on the community feedback, we intend to gradually roll it out to other apps.
```

Comment:
It is also available when creating invoice manually via the UI, and also in the Greenfield API if `amount` is null or undefined.

Reply:
Done https://github.com/btcpayserver/blog/pull/9/commits/aa91931033fcdf9a21a3f9d6d627b289b381cf46